helm-charts/metal3-deploy/charts/ironic/templates/configmap.yaml

```diff
@@ -44,3 +44,5 @@ data:
   IRONIC_USE_MARIADB: "true"
   LISTEN_ALL_INTERFACES: "true"
   IRONIC_KERNEL_PARAMS: console=ttyS0
+  MARIADB_URL_IRONIC: "10.43.255.253/ironic"
```

Comment:
Is `10.43.255.253` always static?

Reply:
This is based on the entries in the metal3-deploy values.yaml.

Reply:
I see. Thanks.